stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -641,7 +641,7 @@ static String createQuery(@Nullable Map<String, Object> params)
     }
 
     @VisibleForTesting
-    static String getApiUrl() {
+    static String getTokensUrl() {
```

Comment:
why was this change made?

Reply:
The method name was misleading. This method returns `"https://api.stripe.com/v1/tokens"`, so it's the tokens URL, not the general API URL. Compare with `getSourcesUrl()` on line 651.

Reply:
cool, thanks!